examples/e-commerce/index.html

```diff
@@ -48,45 +44,72 @@
         Stop looking for an item — find it.
       </p>
 
-      <div id="searchbox"></div>
+      <div data-widget="searchbox"></div>
```

Comment:
is there a reason for not using ids?
ids should probably be the best practice we push for, as they are often faster by x100
https://jsperf.com/id-vs-class-vs-data-attribute/6

Not a blocker at all.

Reply:
fyi https://github.com/algolia/instantsearch.js/pull/3866#issuecomment-504025276

Reply:
@tkrugg See the link above sent by @eunjae-lee.

Speed shouldn't be an issue here. I tried to make the code more descriptive with this approach. Not entirely satisfied but I found it easier to understand where the widgets are injected. I can change if you have concerns.